.github/workflows/update-azure-devops-templates-from-upstream.yaml

```diff
@@ -0,0 +1,30 @@
+# Update the main branch to what upstream has on an hourly basis (and when triggered manually).
+# This is to ensure local copies always match upstream as close as possible.
+name: update-from-upstream
+
+on:
+  schedule:
+    - cron: "0 * * * *" # every hour
+  workflow_dispatch:
```

Comment:
I think it is good practice to add an explicit `nil`.

Reply:
Done